.travis.yml

```diff
@@ -21,3 +21,11 @@ script:
 after_success:
     - "pip install coveralls"
     - coveralls
+deploy:
+    provider: pypi
+    user: edx
+    password:
+        secure: foo
+    distributions: "sdist"
```

Comment:
this is what our bok-choy instructions specify. I was ruling out other distributions with this change (the example on the wiki also lists `bdist_wheel`).